metadata_example.yaml

```diff
@@ -1,10 +1,10 @@
 institution: University of California, San Francisco
 lab: Berke Lab
 experimenter:
-  - Crater, Stephanie
-  - Figueroa, Jose
-  - Hwang, Yang-Sun
-  - Krausz, Tim
+  - Stephanie Crater
+  - Jose Figueroa
+  - Yang-Sun Hwang
+  - Tim Krausz
```

Comment:
DANDI requires experimenter names to be in the format `Last Name, First Name [Middle Initial]` so this change would require reformatting the name internally.

Reply:
I mean - it would require jdb-to-nwb to reformat the names before writing them to the NWB file.

Reply:
When I had converted some nwbs previously, it worked fine for names First Last format. I initially made this change because spyglass had trouble ingesting names that were put into our nwbs in Last, First format (which has now been fixed, so going back to Last, First should be fine). Bit I think either should work?

Reply:
You are right! I was mistaken. Having experimenter names be in the format `Last Name, First Name [Middle Initial]` is an NWB best practice, not an NWB or DANDI rule. My bad.

Either will work, but I recommend that when the data are written to NWB, they follow the best practices.

Reply:
sounds good - given that this has been changed in spyglass to work with Last, First I'll change it back to match best practices :)

Reply:
Thanks! Standardization here helps systems automatically map the string of a person name to datasets in the broader literature / data ecosystem. (ORCID would be even better but we're working on it)